### ISSUE TITLE
Possible regression related to filters in version 0.1.15.

### ISSUE THREAD

**Issue by TrevorS** [thestorefront/FastAPI#5]
Since commit 741a0dcaba03a22e4975a2869f98e6fb59c6a3e6, the example default filter no longer works:
```ruby
# example filter
fastapi_default_filters({
  marbles: {
    radius__lte: 10
  }
})
```
Errors out with:
```
Started GET "/api/v1/buckets" for ::1 at 2015-02-09 14:10:51 -0500
Processing by Api::V1::BucketsController#index as JSON
   (0.3ms)  SELECT COUNT(id) FROM buckets
   (0.7ms)  SELECT buckets.id as id, buckets.color as color, buckets.material as material, people.id as person__id, people.name as person__name, people.gender as person__gender, people.age as person__age, ARRAY_TO_STRING(ARRAY( SELECT ROW( __marbles.id, __marbles.color, __marbles.radius ) FROM marbles as __marbles WHERE __marbles.bucket_id IS NOT NULL AND __marbles.bucket_id = buckets.id AND  ORDER BY __marbles.created_at DESC ), ',') as __many__marbles FROM buckets LEFT JOIN people AS people ON people.id = buckets.person_id  ORDER BY buckets.created_at DESC LIMIT 500 OFFSET 0
PG::SyntaxError: ERROR:  syntax error at or near "ORDER"
LINE 1: ...OT NULL AND __marbles.bucket_id = buckets.id AND  ORDER BY _...
                                                             ^
: SELECT buckets.id as id, buckets.color as color, buckets.material as material, people.id as person__id, people.name as person__name, people.gender as person__gender, people.age as person__age, ARRAY_TO_STRING(ARRAY( SELECT ROW( __marbles.id, __marbles.color, __marbles.radius ) FROM marbles as __marbles WHERE __marbles.bucket_id IS NOT NULL AND __marbles.bucket_id = buckets.id AND  ORDER BY __marbles.created_at DESC ), ',') as __many__marbles FROM buckets LEFT JOIN people AS people ON people.id = buckets.person_id  ORDER BY buckets.created_at DESC LIMIT 500 OFFSET 0
Completed 200 OK in 24ms (Views: 0.2ms | ActiveRecord: 2.8ms)
```

This is the SQL that was generated at commit d9fa7b23fcbd42e0d91de27b0c096f1c754ab923 for the same default filter:
```
Started GET "/api/v1/buckets" for ::1 at 2015-02-09 14:14:34 -0500
Processing by Api::V1::BucketsController#index as JSON
   (0.2ms)  SELECT COUNT(id) FROM buckets
   (0.7ms)  SELECT buckets.id as id, buckets.color as color, buckets.material as material, people.id as person__id, people.name as person__name, people.gender as person__gender, people.age as person__age, ARRAY_TO_STRING(ARRAY( SELECT ROW( __marbles.id, __marbles.color, __marbles.radius ) FROM marbles as __marbles WHERE __marbles.bucket_id IS NOT NULL AND __marbles.bucket_id = buckets.id AND __marbles.radius <= '10' ORDER BY __marbles.created_at DESC ), ',') as __many__marbles FROM buckets LEFT JOIN people AS people ON people.id = buckets.person_id  ORDER BY buckets.created_at DESC LIMIT 500 OFFSET 0
Completed 200 OK in 4ms (Views: 0.1ms | ActiveRecord: 0.9ms)
```

Filters specified in the URL continue to work, for example:
```bash
➜  fast-api-project git:(master) ✗ curl -i http://localhost:3000/api/v1/buckets/\?color\=red
HTTP/1.1 200 OK
X-Frame-Options: SAMEORIGIN
X-Xss-Protection: 1; mode=block
X-Content-Type-Options: nosniff
Content-Type: application/json; charset=utf-8
Etag: W/"93b094ff833a9f1699fde1d7ba52999f"
Cache-Control: max-age=0, private, must-revalidate
X-Request-Id: 3028c5c9-2cd5-4fc5-9479-8555de2b41a6
X-Runtime: 0.005019
Server: WEBrick/1.3.1 (Ruby/2.1.5/2014-11-13)
Date: Mon, 09 Feb 2015 19:13:02 GMT
Content-Length: 365
Connection: Keep-Alive

{"meta":{"total":1,"offset":0,"count":1,"error":null},"data":[{"id":7,"color":"red","material":"wood","person":{"id":4,"name":"Shannon","gender":"female","age":16},"marbles":[{"id":31,"color":"black","radius":2},{"id":32,"color":"yellow","radius":3},{"id":33,"color":"black","radius":1},{"id":34,"color":"blue","radius":5},{"id":35,"color":"orange","radius":10}]}]}
```

I've tested this using Rails 3.2.17 and Rails 4.2.0.

**Comment by keithwhor** [comment]
Nice catch, I'll look into this as soon as I can.

(This is why FastAPI needs tests. :) If you know anybody willing to help, that would be fantastic!)

- Keith

**Comment by TrevorS** [comment]
This is tested for in #9.

**Comment by TrevorS** [comment]
Fixed in #14 and faf5549b036d06ce9c336a7c8613a64e8e3bb809.
